extensions/amp-onetap-google/validator-amp-onetap-google.protoascii

```diff
@@ -0,0 +1,42 @@
+#	
+# Copyright 2020 The AMP HTML Authors. All Rights Reserved.	
+#	
+# Licensed under the Apache License, Version 2.0 (the "License");	
+# you may not use this file except in compliance with the License.	
+# You may obtain a copy of the License at	
+#	
+#      http://www.apache.org/licenses/LICENSE-2.0	
+#	
+# Unless required by applicable law or agreed to in writing, software	
+# distributed under the License is distributed on an "AS-IS" BASIS,	
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.	
+# See the License for the specific language governing permissions and	
+# limitations under the license.	
+#	
+
+tags: {  # amp-onetap-google
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-onetap-google"
+    version: "0.1"
+    version: "latest"
+    requires_usage: ERROR
+  }
+  attr_lists: "common-extension-attrs"	
+}
+
+tags: {  # <amp-onetap-google>
+  html_format: AMP
+  tag_name: "AMP-ONETAP-GOOGLE"
+  requires_extension: "amp-onetap-google"
+  # data-* is generally allowed, but it's not generally mandatory.
+  attrs: {
+    name: "data-src"
+    mandatory: true
```

Comment:
is there a reason not to treat this as a `value_url` and do some validation around protocols?

Reply:
Added `value_url` spec.